src/GitHub.App/Api/ApiClient.cs

```diff
@@ -84,7 +84,7 @@ public IObservable<UserAndScopes> GetUser()
             }
             else
             {
-                log.Error($"Error reading scopes: /user succeeded but {ScopesHeader} was not present.");
+                log.Error("Error reading scopes: /user succeeded but {ScopesHeader} was not present.", ScopesHeader);
```

Comment:
Or maybe `Log.Error` is doing something magical with the parameter name that I'm not familiar with?